applications/madini_tracechain.md

```diff
@@ -129,37 +170,75 @@ Project Details
 
 We expect the teams to already have a solid idea about the project's expected final state. Therefore, we ask the teams to submit (where relevant):
 
-    Mockups/designs of any UI components
-    API specifications of the core functionality
-    An overview of the technology stack to be used
-    Documentation of core components, protocols, architecture, etc. to be deployed
```

Comment:
TraceChain interfaces with both block-chain and a server with a database. Madini TraceChain relies on the Vechain blockchain a distributed block-chain service. Verified data is broadcasted to the blockchain and stored as hex data. Madini TraceChain relies on the Vechain blockchain a distributed block-chain service.Verified data is broadcasted to the blockchain and stored as hex data also stores the raw data to the SQLite 

Madini TraceChain Technical Details:

•Python 3.8 (Flask)
•VPS / Server.
•Sqlite Database.
•VET Wallet address with VTHO.
•VIP 190

TraceChain consists of Web App, API’s .The Web application is made using JQUERY,JAVASCRIPT and python 3.8 flask which integrates VIP 191 which help send data to the Vechain blockchain as well as saving data to the database.

How does VIP-191 Work?

VIP-191 comes with fee delegation protocol on VeChain.VIP-191 works by delegating a secondary address to pay the transaction cost. When processing, the transaction will be signed by the secondary address, before being sent to the blockchain.VIP-191 is using a specially formed signature composed by the user and the sponsor together.
Just as the picture below has depicted, once the transaction is forged, the user extracts the hash (information digest) and signs it as normal. At the same time, the sponsor also signs the hash but with additional information “for whom”. Then we concatenate the two signatures together, then send the final composed signature along with the transaction itself, to the VeChain network.

Ref: Architecture

VIP 191 is an upgrade to the VeChain core blockchain. Initially proposed by Totient, it allows users to bypass a transaction fee (VTHO) by delegating it to an application they are using.